docs/_source/getting_started/installation/deployments/huggingface-spaces.md

```diff
@@ -189,3 +189,60 @@ Additionally, the `LOAD_DATASETS` will let you configure the sample datasets tha
     1. `single`: Load single datasets for TextClassification task.
     2. `full`: Load all the sample datasets for NLP tasks (TokenClassification, TextClassification, Text2Text)
     3. `none`: No datasets being loaded.
+
+## Setting up HF Authentication
+
+From version 1.23.0 you can enable Hugging Face authentication for your Argilla Space. This feature allows you to give access to your Argilla Space to users that are logged in to the Hugging Face Hub.
+
+```{note}
+This feature is specially useful for public crowdsourcing projects. If you would like to have more control over who can log in to the Space, you can set this up on a private space so that only members of your Organization can sign in. Alternatively, you may want to [create users](/getting_started/installation/configurations/user_management.md#create-a-user) and use their credentials instead.
+```
+
+To enable this feature, you will first need to [create an OAuth App in Hugging Face](https://huggingface.co/docs/hub/oauth#creating-an-oauth-app). To do that, go to your user settings in Hugging face and select *Connected Apps* > *Create App*. Once inside, choose a name for your app and complete the form with the following information:
+
+* **Homepage URL:** [Your Argilla Space Direct URL](/getting_started/installation/deployments/huggingface-spaces.md#your-argilla-space-url).
+* **Logo URL:** `https://[your_space_direct_url]/favicon.ico`
+* **Scopes:** `openid` and `profile`.
+* **Redirect URL:** `https://[your_space_direct_url]/oauth/huggingface/callback`
+
+This will create a Client ID and an App Secret that you will need to add as variables of your Space. To do this, go to the Space *Settings* > *Variables and Secrets* and save the Client ID and App Secret as environment secrets like so:
+
+1. Name: `OAUTH2_HUGGINGFACE_CLIENT_ID` - Value: [Your Client ID]
+2. Name: `OAUTH2_HUGGINGFACE_CLIENT_SECRET` - Value: [Your App Secret]
```

Comment:
```suggestion
1. **Name:** `OAUTH2_HUGGINGFACE_CLIENT_ID` - **Value:** [Your Client ID]
2. **Name:** `OAUTH2_HUGGINGFACE_CLIENT_SECRET` - **Value:** [Your Client Secret]
```

Reply:
I believe this one is called "App Secret" in the Connected Apps settings in Hugging Face. That's why I chose that naming, but let me know if it's not clear that way.